### PR TITLE
Document --benchmark_filter

### PR DESCRIPTION
**Pull request by NAThompson**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -463,3 +463,20 @@ cmake -DCMAKE_BUILD_TYPE=Release -DBENCHMARK_ENABLE_LTO=true
 
 ## Linking against the library
 When using gcc, it is necessary to link against pthread to avoid runtime exceptions. This is due to how gcc implements std::thread. See [issue #67](https://github.com/google/benchmark/issues/67) for more details.
+
+
+## Running a subset of the benchmarks
+
+Similar to the `--gtest_filter` functionality of `googletest`, `benchmark` has a `--benchmark_filter` option:
```
Comment (pleroy) [inline README.md]: It's not so similar because the two take different kinds of regexp.  Please document this.

Comment (NAThompson) [inline README.md]: I don't think anyone cares what sort of regex the filter takes; seems a little detailed for a README. But nonetheless, if the analogy is confusing, why don't we just write:

> To run a subset of the benchmarks, use the `--benchmark-filter` option:

and forget about googletest?

Comment (dominichamon) [inline README.md]: that sounds great.

**Comment by EricWF** [comment]
LGTM. Please resolve merge conflicts.

**Comment by EricWF** [comment]
Closing due to inactivity. I committed similar documentation in 9c261681266d051c0b04ddbe2064b569c8b19528.

**Pull request closed without merging**